### DISCUSSION TITLE
root image path is not accessible after cloudimage thumbnail pattern update

### DISCUSSION THREAD

**Discussion by paraskumar12011996** [FriendsOfShopware/FroshPlatformThumbnailProcessor#143] [Q&A]
Hello @tinect ,
Good morning!

I am using the FroshPlatformThumbnailProcessor with cloudimage and it is working fine.
I have also one IDEALO product feed within shop.

And in FroshPlatformThumbnailProcessor plugin configuration only main sales channel is configured with pattern settings and for rest sales channel and product feed it is not active.

So the image is only accessible on this path  https://TOKEN.cloudimg.io/www.myshop.com/media/01/01/01/tinect.png?w=800&func=bound&org_if_sml=1 

and on root path https://www.myshop.com/media/01/01/01/tinect.png I am not able to access the image.

So is their any option, Because I want to access the image on both ways. optimised image as well as real image.

Thank you.

**Comment by tinect** [comment]
> And in FroshPlatformThumbnailProcessor plugin configuration only main sales channel is configured with pattern settings and for rest sales channel and product feed it is not active.

Then the general setting is used. there is no way, to set it per saleschannel. limited by shopware.

> So the image is only accessible on this path https://token.cloudimg.io/www.myshop.com/media/01/01/01/tinect.png?w=800&func=bound&org_if_sml=1
and on root path https://www.myshop.com/media/01/01/01/tinect.png I am not able to access the image.

This sounds strange while the first one is just taking the image from the second path. Do you use any external storage? ou might want to check the connection and if you might have removed the image.

**Comment by paraskumar12011996** [comment]
Hello @tinect 
![image](https://github.com/user-attachments/assets/ae558722-8d0c-4ad8-9cbf-10f38d2e14db)
Please refer to the above image and this shown configuration is updated for Alle Verkaufskanäle
And only working sales channel is configured with Thumbnail-pattern (Cloudimage) 
![image](https://github.com/user-attachments/assets/134d0cc5-031e-4329-bcf7-13cb1d89b1fb)

But unfortunately I am not able to access the root utr path of image and image is available in media and not deleted.

Thank you.

**Reply by tinect** [reply]
I’m sorry, but I have no idea regarding your issue while it sounds not logic to me. It sounds like, anyone should check your cloudimage and webserver settings.
